CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.2.0] - 2019-05-21
+
```

Comment:
you forgot to modify to bottom of the file.

Reply:
indeed, sorry